src/ConfigResource.php

```diff
@@ -51,6 +58,16 @@ public function __construct(array $config, $fileName, ConfigWriter $writer)
         $this->writer   = $writer;
     }
 
+    /**
+     * Enables or disables the sorting of the config data.
+     *
+     * @param bool $enabled
+     */
+    public function setSortingEnabled($enabled)
+    {
+        $this->sortingEnabled = $enabled;
```

Comment:
Cast this value to `(bool)` when assigning.

Reply:
Yes, that's better, changed. Type hinting would be even better of course ;)